tests/installation/addon_products_sle.pm

```diff
@@ -147,7 +147,7 @@ sub test_addonurl {
 sub run {
     my ($self) = @_;
 
-    if (get_var('SKIP_INSTALLER_SCREEN', 0)) {
+    if (get_var('SKIP_INSTALLER_SCREEN', 0) && !get_var('UPGRADE', 0)) {
```

Comment:
Why add this condition?

Reply:
cause this is a migration with MU it will match two `if` in this file and will match `inst-addon` 2 times.

Reply:
Ok, then I suggest to add the condition to next line: advance_installer_window('inst-addon');, so that we will set the SKIP_INSTALLER_SCREEN to 0 as MU process expected.

Reply:
changed.